kernel/configs/fragments/common/network.conf

```diff
@@ -31,8 +31,8 @@ CONFIG_IP_PNP=y
 CONFIG_IP_PNP_DHCP=y
 CONFIG_SYN_COOKIES=y
 CONFIG_TCP_CONG_ADVANCED=y
-CONFIG_TCP_CONG_BBR=y
```

Comment:
AFAIU, can we just leave BBR enabled, but enable CUBIC and make it default?

May make it a bit easier for folks who want to switch?